src/modules/assessments/components/AssessmentFormActions.tsx

```diff
@@ -0,0 +1,103 @@
+import { Stack } from '@mui/material';
+import { useMemo } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import DeleteAssessmentButton from './DeleteAssessmentButton';
+import PrintViewButton from '@/components/layout/PrintViewButton';
+import AssessmentAutofillButton from '@/modules/assessments/components/AssessmentAutofillButton';
+
+import { EnrollmentDashboardRoutes } from '@/routes/routes';
+import {
+  AssessmentRole,
+  EnrollmentFieldsFragment,
+  FullAssessmentFragment,
+} from '@/types/gqlTypes';
+import { generateSafePath } from '@/utils/pathEncoding';
+
+interface Props {
+  enrollment: EnrollmentFieldsFragment;
+  assessment?: FullAssessmentFragment;
+  onAutofill: VoidFunction;
+  isPrintView: boolean;
+  locked: boolean;
+  showAutofill?: boolean;
+}
+
+const AssessmentFormActions: React.FC<Props> = ({
```

Comment:
This code isn't new, it is what became of `AssessmentFormSidebar` after I factored the `FormStepper` out into `FormNavigationContainer`.